docs/reference/technology/connectivity/cellular.md

```diff
@@ -0,0 +1,159 @@
+
+### Quick start
+
+There are two phases to Mbed OS connectivity:
+
+1. Connect to a network.
+1. Open a socket to send or receive data.
+
+The easiest way to connect your application to the internet over a cellular network is to use the `CellularConnectionFSM` class. It encapsulates most of the complexity of connecting to the cellular network and also reports any changes in connection status to your application. When connected to a cellular network, you can use Mbed OS network sockets as usual, as Figure 2 illustrates.
+
+<span class="images">![](https://s3-us-west-2.amazonaws.com/mbed-os-docs-images/api-cellular-quick-start.png)<span>Figure 2. Connect to cellular network and open a socket</span></span>
+
+If you want to see code, you can go to our [cellular example](https://github.com/ARMmbed/mbed-os-example-cellular) or 
+our [advanced cellular example](https://github.com/ARMmbed/mbed-os-example-cellular-advanced).
+
+### Cellular connection management
+
+Applications can use cellular APIs to fine-control the cellular connection. Cellular APIs are structured based on main functionalities:
+
+- `CellularNetwork` for cellular network features, such as preferred operator and APN.
+- `CellularPower` for cellular module power control, such as enabling power save.
+- `CellularInformation` to read the cellular module type and firmware version.
+- `CellularSIM` to enter the PIN code and other SIM management functions.
+- `CellularSMS` to read and write SMS messages.
+
+The CellularDevice class encloses cellular APIs. Therefore, to use any cellular API, you need to get CellularDevice first. You can then use CellularDevice to open and close cellular APIs, as Figure 3 illustrates.
+
+<span class="images">![](https://s3-us-west-2.amazonaws.com/mbed-os-docs-images/api-cellular-device.png)<span>Figure 3. Use CellularDevice to open Cellular APIs</span></span>
+
+When an application has opened a cellular API, you can use it to request API methods. For example:
+
+    CellularNetwork *network  = cellularDevice->get_network();
+    if (network) {
+        printf("Local IP address is %s", network->get_ip_address());
+    }
+
+### UDP and TCP sockets
+
+If you want to use UDP or TCP sockets, you need an IP stack. Mbed OS cellular has an option to use either the LWIP stack, which is part of Mbed OS, or to use the IP stack on the cellular module. Figure 4 illustrates IP stack deployment.
+
+<span class="images">![](https://s3-us-west-2.amazonaws.com/mbed-os-docs-images/api-cellular-ip-stack.png)<span>Figure 4. IP stack can be used in PPP or AT mode</span></span>
+
+#### PPP mode with the LWIP stack on Mbed OS
+
+In PPP mode, the LWIP stack is linked as a part of the Mbed OS application. Sockets are implemented on the LWIP stack, and the modem is used over a Point-to-Point Protocol (PPP) link as a plain data pipe.
```

Comment:
Query: Who or what implements sockets on the LWIP stack? You, the user? Or something else? Also, who or what uses the modem over a PPP link?

Reply:
Updated description about IP stack selection in https://github.com/AriParkkila/Handbook/commit/df8b05aa9ec9438fde316659a9aa243b5bb3c66e

Fixed also "AT commands are modem specific, and you need to implement them" due to this page is intended to application developers... who are not the ones to make porting.